app/controllers/maps_controller.rb

```diff
@@ -86,6 +86,20 @@ def usermaps
     end
   end
 
+  # GET maps/new
+  def new
+    @map = Map.new(:name => "Untitled Map", :permission => "public", :arranged => true)
+    authorize @map, :create?
```

Comment:
You can just say authorize @map 

app/policies/application_policy.rb defines this function:

    def new?
      create?
    end